docs/source/main_classes/trainer.rst

```diff
@@ -113,7 +113,117 @@ Toward Training Trillion Parameter Models, by Samyam Rajbhandari, Jeff Rasley, O
 
 This provided support is new and experimental as of this writing.
 
-You will need at least 2 GPUs to benefit from these features.
+Installation Notes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As of this writing, both, FairScale and Deepspeed, require compilation of CUDA C++ code, before they can be used.
+
+While all installation issues should be dealt with through the corresponding github Issues of FairScale and DeepSpeed,
+there are a few common issues that one may encounter while building any pytorch extension that needs to build CUDA
+extensions.
+
+Therefore, if you encounter a CUDA-related build issue while doing one of the following or both:
+
+.. code-block:: bash
+
+    pip install fairscale
+    pip install deepspeed
+
+please, read the following notes first.
+
+In these notes we give examples for what to do when ``pytorch`` has been built with CUDA ``10.2``. If your situation is
+different remember to adjust the version number to the one you are after.
+
+**Possible problem #1:**
+
+While, pytorch comes with its own CUDA toolkit, to build these two projects you must have an identical version of CUDA
```

Comment:
```suggestion
While, PyTorch comes with its own CUDA toolkit, to build these two projects you must have an identical version of CUDA
```
(Everywhere in text pytorch -> PyTorch, the \`\`pytorch\`\` in backsticks can stay the same, will stop highlighting)